spec/classes/puppet_server_passenger_spec.rb

```diff
@@ -6,10 +6,15 @@
     next if exclude_test_os() and exclude_test_os.include?(os)
     next if os_facts[:osfamily] == 'windows'
     context "on #{os}" do
+      let(:pre_condition) do
+        'include ::puppet'
+      end
```

Comment:
Yep, just edit the `default_params` let block below and add :passenger_pre_start, :port and :http_port.  This saves us from loading the entire module in, which should make a big difference to the speed of the tests.